prog/weaveexec/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM alpine
+#FROM alpine
+FROM jeanblanchard/alpine-glibc
```

Comment:
The calico plugin (and I image a number of other CNI plugins too) have a dependency on glibc.

(Obviously the commented out line should be removed)